tests/fieldmodule/fieldassignment.cpp

```diff
@@ -107,9 +107,9 @@ TEST(ZincFieldassignment, cubeOffsetScale)
 
     EXPECT_EQ(RESULT_OK, zinc.root_region.readFile(resourcePath("fieldmodule/cube.exformat").c_str()));
 
-	FieldNodeGroup nodeGroup = zinc.fm.createFieldNodeGroup(nodes);
+	FieldGroup nodeGroup = zinc.fm.createFieldGroup();
```

Comment:
Probably a misleading variable name here, 'nodeGroup'  is a bit of a hangover from previous API.

Reply:
Fixed.